changes/ce/feat-12561.en.md

```diff
@@ -1,21 +1,20 @@
-Implement HTTP APIs to get the list of client's inflight and mqueue messages.
+Implement HTTP APIs to get the list of client's in-flight and mqueue messages.
 
 To get the first chunk of data:
  - GET /clients/{clientid}/mqueue_messages?limit=100
  - GET /clients/{clientid}/inflight_messages?limit=100
 
 Alternatively:
- - GET /clients/{clientid}/mqueue_messages?limit=100&after=none
- - GET /clients/{clientid}/inflight_messages?limit=100&after=none
+ - GET /clients/{clientid}/mqueue_messages?limit=100&position=none
```

Comment:
As this is unreleased feature updated, I think it's fine to change the previous change log instead of creating a new one...